### PR TITLE
Task portfolio

### PR DESCRIPTION
**Pull request by HadertS**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h1>Placeholder Text</h1>
 			<section id='about'>
 				<figure>
 					<img alt='Sammi Hadert and Mouse the cat' src='img/author 90.jpg'>
```
Comment (Steph-ski) [inline index.html]: HTML validator brought this up as an error: Bad value img/author 90.jpg for attribute src on element [img](https://html.spec.whatwg.org/multipage/#the-img-element): Illegal character in path segment: space is not allowed.

File: index.html [hunk 2]
```diff
@@ -40,8 +40,57 @@ <h1>Hi, I’m Sammi!</h1>
...
+						<button type='button'>Code</button> 
+					</article>
+				</div>
+			</section>
+			<section id='contact'>
+				
+			</section>
 		</main>
 		<footer>
 			<p>&#169 Sammi Hadert 2023</p>
```
Comment (Steph-ski) [inline index.html]: HTML validator brought this up as an error: Character reference was not terminated by a semicolon.

File: index.html [hunk 3]
```diff
@@ -40,8 +40,57 @@ <h1>Hi, I’m Sammi!</h1>
 					<p>Oh no its even stuff about me</p>
 				</article>
```
Comment (Steph-ski) [inline index.html:41]: Consider adding a Portfolio heading?

File: styles.css [hunk 4]
```diff
@@ -128,24 +127,64 @@ section {
...
 	}
+	
 	#about img {
 		  width: 100%;
 	}
+	
+	#portfolio div {
+		display: block;
+	}
+	∂ßß
```
Comment (Steph-ski) [inline styles.css]: Should this be here?

**Review by Steph-ski — COMMENTED** [review]
Clear code.

When moving to mobile view, the portfolio boxes don't space out vertically.

2 errors picked up on HTML validator. 

All CSS values used consistently (rem & Hex codes).

File: styles.css (hunk 4 above)
Comment (HadertS) [reply styles.css]: removed

File: index.html (hunk 2 above)
Comment (HadertS) [reply index.html]: semicolon added

File: index.html (hunk 1 above)
Comment (HadertS) [reply index.html]: space changed to '-' here and image filename

File: index.html (hunk 3 above)
Comment (HadertS) [reply index.html:41]: added

**Review by Steph-ski — APPROVED** [review]

**Pull request merged**